BaseApp/COD50001.TXT

```diff
@@ -0,0 +1,49 @@
+OBJECT Codeunit 50001 Path Helper
```

Comment:
is it normal to add CDUs to 50xxx? isn't that what most people use to add customer stuff?

Reply:
From https://github.com/Microsoft/cal-open-library/blob/master/README.md
> 3. Object IDs should be created within the range 50K-99K.

Reply:
We're going to change them when processing the request.

Reply:
Code accepted. I will add the requested functions to COD419. Thanks for your contribution!

Reply:
Requested functions were added to COD419. Actually, most of the requested functions already existed, but weren't exposted. They are now. For all changes, please refer to #39.